packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -142,6 +142,21 @@ export const API_GUIDES = {
   SG: 'https://docs.ovh.com/sg/en/api/first-steps-with-ovh-api/',
 };
 
+export const TERRAFORM_GUIDES = {
+  DEFAULT: 'https://docs.ovh.com/gb/en/publiccloud/databases/order-terraform/',
+  ASIA: 'https://docs.ovh.com/asia/en/publiccloud/databases/order-terraform/',
+  AU: 'https://docs.ovh.com/au/en/publiccloud/databases/order-terraform/',
+  CA: 'https://docs.ovh.com/ca/en/publiccloud/databases/order-terraform/',
+  FR: 'https://docs.ovh.com/fr/publiccloud/databases/order-terraform/',
+  GB: 'https://docs.ovh.com/gb/en/publiccloud/databases/order-terraform/',
+  IE: 'https://docs.ovh.com/ie/en/publiccloud/databases/order-terraform/',
+  QC: 'https://docs.ovh.com/ca/fr/publiccloud/databases/order-terraform/',
+  SG: 'https://docs.ovh.com/sg/en/publiccloud/databases/order-terraform/',
+};
+
+export const DATABASE_CREATION =
```

Comment:
Can you use a more explicit name, like "guides" or "exemples" ? This naming is a bit confusing imo.

Reply:
Fixed :)